test/compress/comparing.js

```diff
@@ -4,59 +4,40 @@ keep_comparisons: {
         unsafe_comps: false
     }
     input: {
-        var obj1 = {
-            valueOf: function() {triggeredFirst();}
-        }
-        var obj2 = {
-            valueOf: function() {triggeredSecond();}
-        }
+        var obj1, obj2;
```

Comment:
(I can't seem to put a line comment on an unchanged line, so this will have to do.)

The test is currently named `keep_comparisons`, which no longer reflects the expected result. Consider renaming it. 😉

Reply:
I've updated this test to consider name 😌